api/app/src/mappings/apple/index.ts

```diff
@@ -16,35 +19,36 @@ export type AppleWebhookPayload = {
   sleep?: Sleep[];
 };
 
-export function mapData(data: AppleHealth): AppleWebhookPayload {
+export function mapData(data: AppleHealth, hourly: boolean): AppleWebhookPayload {
   const payload: AppleWebhookPayload = {};
 
-  const activityData = mapDataToActivity(data);
+  const activityData = mapDataToActivity(data, hourly);
 
   if (activityData.length) payload.activity = activityData;
 
-  const bodyData = mapDataToBody(data);
+  const bodyData = mapDataToBody(data, hourly);
 
   if (bodyData.length) payload.body = bodyData;
 
-  const biometricsData = mapDataToBiometrics(data);
+  const biometricsData = mapDataToBiometrics(data, hourly);
 
   if (biometricsData.length) payload.biometrics = biometricsData;
 
-  const nutritionData = mapDataToNutrition(data);
+  const nutritionData = mapDataToNutrition(data, hourly);
 
   if (nutritionData.length) payload.nutrition = nutritionData;
 
-  const sleepData = mapDataToSleep(data);
+  const sleepData = mapDataToSleep(data, hourly);
 
   if (sleepData.length) payload.sleep = sleepData;
 
   return payload;
 }
 
-export const createMetadata = (date: string) => {
+export const createMetadata = (date: string, hourly: boolean): Metadata => {
   return {
-    date,
+    date: dayjs(date).format(ISO_DATE),
+    ...(hourly ? { hour: dayjs(date).format("HH:mm:ss") } : undefined),
```

Comment:
If this signifies the hour, should we just have this to always be `HH:00`?

mm:ss might be confusing, especially if they're diff between payloads